_topic_maps/_topic_map_osd.yml

```diff
@@ -301,8 +299,80 @@ Name: Authentication and authorization
 Dir: authentication
 Distros: openshift-dedicated
 Topics:
+- Name: Authentication and authorization overview
+  File: index
+- Name: Understanding authentication
+  File: understanding-authentication
+# - Name: Configuring the internal OAuth server
```

Comment:
Should these commented out blocks be removed?

Reply:
I've been commenting them out so we can add them if we ever need to.